### ISSUE TITLE
Missing entries in BUFRCREX CodeFlag

### ISSUE THREAD

**Issue by SibylleK** [wmo-im/BUFR4#73]
If the unit of a table B entry is "Common Code table C-..", the entry in Code tables and Flag table contains the name and a reference like "(See common Code table C-1 in Part C/c.)".  (Unfortunately the reference only exist in the PDF-documents.)

But, the entries for 001035 - "Originating/generating centre" (Common code table C-11) and 008046 - "Atmospheric chemical or physical constituent type" (Common code table C-14) are completely missing.

**Comment by amilan17** [comment]
@SibylleK -- looks like some manual and repository cleanup is needed, but I don't fully understand. Can you add some examples or references to pages in this issue?

**Comment by SibylleK** [comment]
Dear @amilan17, 
the pages which I mean are I.2 –   CODE/FLAG Tables/01 — 2 (i.e. page 620 of the pdf document) and I.2 –   CODE/FLAG Tables/08 — 17 (i.e. page 697).
And the missing entries would be:  

0 01 035
Originating/generating centre
(See common Code table C–11 in Part C/c.)

and

0 08 046
Atmospheric chemical or physical constituent type
(See common Code table C–14 in Part C/c.)

**Comment by amilan17** [comment]
https://github.com/wmo-im/BUFR4/compare/issue73

**Comment by amilan17** [comment]
@jitsukoh - this looks like a simple update to the CSV and the manual. If you agree, we can include this in FT2021-2.

**Comment by jitsukoh** [comment]
@amilan17 @SibylleK I agree with the inclusion of this issue to FT2021-2. I move this to in validation status.
The branch is updated but there is a small inconsistency in upper/lower cases:

> 001033,Identification of originating/generating centre,,,,,"(See **C**ommon Code table C–1 in Part C/c.)",Operational
> 001035,Originating/generating centre,,,,,"(See **c**ommon Code table C-11 in Part C/c.)",Operational

**Comment by jitsukoh** [comment]
@amilan17 there is a small issue in upper/lower cases in the branch but I think the Secretariat will anyway check and take care of these things, so please move this to validated status when you are ready. From my side, it is done.
